### DISCUSSION TITLE
Checking in on Troubles with Dark Matter

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#80] [Cosmology Models]
This post has a good list of *challenges stemming from observational reality* [deviating from the expectations of ΛCDM](https://tritonstation.com/2023/06/27/checking-in-on-troubles-with-dark-matter/).  The conclusion for most of the examples is that "*this one remains a problem*".

Based on the paper by Famaey and McGaugh, *Modified Newtonian Dynamics (MOND): Observational Phenomenology and Relativistic Extensions* Living Rev. Relativ. 15, 10 (2012). [https://doi.org/10.12942/lrr-2012-10](https://doi.org/10.12942/lrr-2012-10)

So ΛCDM doesn't work, we know that, but can *your model* solve these problems?

**Comment by HanDeBruijn** [comment]
> So ΛCDM doesn't work, we know that, but can _your model_ solve these problems?

No, but I get sick by all those MOND advertising. So I shall throw up the following conjecture.

### MOND is emergent from the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) as has been implemented in [UAC](http://www.alternatievewiskunde.nl/cosmology/wild.htm)

Let's start with a quick review of [MOdified Newtonian Dynamics](https://en.wikipedia.org/wiki/Modified_Newtonian_dynamics#Overview), especially the formulas for the deep-MOND regime ( 
$F=$ force, $m=$ test mass, acceleration $a \ll a_0$ ) :

$$
F_N = m \frac{a^2}{a_0}
$$

Applying this to an object of mass $m$ in circular orbit around a point mass $M$ (a crude approximation for a star in the outer regions of a galaxy), we find:

$$
\frac{GMm}{r^2} = m\frac{(v^2/r)^2}{a_0} \quad \Longrightarrow \quad v^4 = GMa_0
$$

That is, the star's rotation velocity $v$ is independent of $r$, its distance from the centre of the galaxy - the rotation curve is flat, as required. In comparison we have, with [MOND = UAC](https://github.com/orgs/a-cosmology-group/discussions/70#discussioncomment-6265312):

$$
\mbox{observed } u \sim \sqrt{\mbox{distance to nucleus } r/R} \times \mbox{calculated } v
$$

Where our previously dimensionless $0 \le r \le 1$ is replaced by the real world one, by multiplying it with the galaxy's radius $R$.
For the same crude approximation we see that the rotation curve is flat as required, but with a square instead of a fourth power and __no free parameter__ $a_0$ :

$$
\frac{GMm}{r^2} = m\frac{v^2}{r} \quad \Longrightarrow \quad v^2 = \frac{GM}{r}
$$

$$
u^2 = \frac{r}{R} v^2 \quad \Longrightarrow \quad u^2 = \frac{GM}{R}
$$

Let us now calculate [Mordehai Milgrom](https://en.wikipedia.org/wiki/Mordehai_Milgrom)'s factor $a_0$ :

$$
\left(u^2\right)^2 (\mbox{UAC}) = v^4 (\mbox{MOND}) \quad \Longrightarrow \quad \left(\frac{GM}{R}\right)^2 = GMa_0 
\quad \Longrightarrow \quad \boxed{ a_0 = \frac{GM}{R^2} }
$$

So Milgrom's constant - as it appears in UAC - is equal to the standard gravitational parameter divided by the radius squared of a galaxy.
